toucan_connectors/common.py

```diff
@@ -198,6 +198,13 @@ def transform_with_jq(data: object, jq_filter: str) -> list:
     '<a href="https://stedolan.github.io/jq/manual/">documentation</a>',
 )
 
+XpathSchema = Field(
```

Comment:
Should it be a StringField?

Reply:
I was not sure about that, when I saw
```
   filter: str = FilterSchema
``` 
Defined in common.py, I supposed it would need a specific Schema. But I'm fine to use a StringField

Reply:
I was not sure about that, for FilterSchema it's a Field. But I agree that StringField is more relevant.

Reply:
Are you sure this type exists ? 
![image](https://user-images.githubusercontent.com/71276735/96580979-682e4c80-12d9-11eb-8f45-f9a25cc15efb.png)

Reply:
Sorry, I took the naming from MongoEngine, not Pydantic ^^
In Pydantic, there is a `StrictStr` (https://pydantic-docs.helpmanual.io/usage/types/#strict-types) type, but it's OK to use Field as well.
My apology for this unrelevant comment